src/flask_sqlalchemy/__init__.py

```diff
@@ -999,7 +999,11 @@ def _execute_for_all_tables(self, app, bind, operation, skip_tables=False):
             op(bind=self.get_engine(app, bind), **extra)
 
     def create_all(self, bind="__all__", app=None):
-        """Creates all tables.
+        """
+            Creates all tables.
```

Comment:
First line should be on the same line as the `"""`. Additional lines should not be indented.

Reply:
my bad, got that format from [here](https://github.com/pallets/flask-sqlalchemy/blob/master/src/flask_sqlalchemy/__init__.py#L934): 

```python
# __init.py__
    def create_engine(self, sa_url, engine_opts):
        """
            Override this method to have final say over how the SQLAlchemy engine
            is created.
            In most cases, you will want to use ``'SQLALCHEMY_ENGINE_OPTIONS'``
            config variable or set ``engine_options`` for :func:`SQLAlchemy`.
        """
        return sqlalchemy.create_engine(sa_url, **engine_opts)
```

figured it might be a way to handle more verbose docstrings that had more than one line.

Reply:
just updated and re-pushed